master_study/master_jobs/002_tracking_job/000_track.py

```diff
@@ -66,10 +66,10 @@
 # Symplify line #
 #################
 
-line.remove_inactive_multipoles(inplace=True)
-line.remove_zero_length_drifts(inplace=True)
-line.merge_consecutive_drifts(inplace=True)
-line.merge_consecutive_multipoles(inplace=True)
+#line.remove_inactive_multipoles(inplace=True)
+#line.remove_zero_length_drifts(inplace=True)
+#line.merge_consecutive_drifts(inplace=True)
+#line.merge_consecutive_multipoles(inplace=True)
```

Comment:
I cannot add a comment in line 79 so I include the comment her. In line 79, we can add: 

tracker.optimize_for_tracking()

as suggested by Gianni to make the tracking faster